common/util.go

```diff
@@ -342,10 +342,16 @@ func IsServiceHandlerRetryableError(err error) bool {
 		return false
 	}
 
-	switch err.(type) {
+	switch t := err.(type) {
 	case *serviceerror.Internal,
 		*serviceerror.Unavailable:
 		return true
+	case *serviceerror.MultiOperationExecution:
+		for _, opErr := range t.OperationErrors() {
+			if IsServiceHandlerRetryableError(opErr) {
```

Comment:
It looks like opErr is sometimes nil here (see the test failures)? Shouldn't the constructor filter out nils?

Reply:
Right, thank you! I added a nil check.

The reason the constructor doesn't filter out `nil`s is because the errors are positional, ie error index 0 maps to operation index 0. If we filtered them out, the indexes wouldn't line up anymore.